http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -305,32 +296,42 @@ class GKEInterpreter[F[_]: Parallel: ContextShift: Timer](
       gkeClusterId = dbCluster.getGkeClusterId
       googleProject = params.googleProject
 
-      _ <- logger.info(ctx.loggingCtx)(
```

Comment:
Can we keep this log statement?

Reply:
this log was right before logging `Creating namespace` xxx, there's nothing interesting happening between the 2 log lines..so I figured we don't need both of them. I can add `Begin app creation` text to the second line

Reply:
FWIW, when looking through logs having app creation and namespace creation lines separate would make it easier for me to spot things.

Reply:
Am ok with combining the logs.. mainly just wanted "begin" signal in the logs.